modules/gdnative/include/gdnative/gdnative.h

```diff
@@ -286,6 +286,10 @@ void GDAPI godot_print(const godot_string *p_message);
 
 bool GDAPI godot_is_instance_valid(const godot_object *p_object);
 
+//tags used for safe dynamic casting
+void GDAPI *godot_get_class_ptr(const char *p_classname);
```

Comment:
`godot_string_name` is available from GDNative, why not using it instead of doing a conversion withing the body of the function ?

Reply:
Good idea, thanks. Will update this on Wednesday once I'm back at my computer.